### DISCUSSION TITLE
use flowbite css import without tailwind core css

### DISCUSSION THREAD

**Discussion by KinG-InFeT** [themesberg/flowbite#214] [General]
**Describe the bug**
i can't install basic tailwindcss + possibly the necessary classes for flowbite. in addition to this the library integrated in flowbite is one of an obsolete version of tailwindcss (Arrived at 3.1.5) for which it can be updated via npm.

It should be possible to integrate the flowbite custom css without the tailwindcss lib css present, so as to have full control of the css import.

**Comment by zoltanszogyenyi** [comment]
Hello @KinG-InFeT,

Theoretically, Flowbite should work independently from which version you have of Tailwind CSS installed.

It's true that our own docs use an earlier version of TW which we will shortly update, but that shouldn't affect your project when pulling Flowbite.

**Reply by KinG-InFeT** [reply]
unfortunately instead yes, it could dare problems integrating flowbite and tailwind, just create an empty laravel project with only the init of the assets with tailwind, add flowbite and you will see that many things break. the best thing is to be able to insert flowbite a tailwind underwater that could have an installation, so as not to go to conflict, I in my project I have not inserted the css glowbite, but only the js for the functionalities.

**Comment by zoltanszogyenyi** [comment]
Hey @KinG-InFeT,

I went ahead and updated Tailwind CSS to the latest version for the next update.

Can you please show some of the breaking things here?

Thanks!

**Comment by KinG-InFeT** [comment]
@zoltanszogyenyi yes,
i create a empty laravel project and install tailwindcss base
i add base html page with standard template from tailwindui.com , 
this is test page: https://pastebin.com/raw/KEqunwRH
tailwind.config.js: https://pastebin.com/raw/cdaZ4nNv

the correct page is: 
![image](https://user-images.githubusercontent.com/193969/179258775-85fae89b-112d-4fa0-b6c5-e8765b3e2b58.png)

but on my laravel project is:
![image](https://user-images.githubusercontent.com/193969/179258842-38c31d8d-11cd-49db-82c4-563ae2c65bef.png)

(i know flowbite css should not be included after standard tailwind, but it shouldn't affect the base tailwind anyway. the best solution is to separate tailwind core from flowbite's css file in order to have maximum freedom of inclusion.)
